### ISSUE TITLE
New color-contrast-enhanced rule (WCAG AAA)

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3189]
We're increasingly starting to see folks ask for AAA tests. One rule we could easily add is a color-contrast-enhanced rule, which uses 7:1 contrast minimum for regular text, and 4.5:1 for large and bold text. See: https://www.w3.org/TR/WCAG21/#contrast-enhanced

This rule will be a copy of the existing color-contrast rule and check, using different check option defaults.

This rule will have to be off by default. Whether or not we do this for all AAA rules is yet to be decided. Currently the only AAA rule we have is on by default, but for something like color-contrast, it's probably not reasonable to keep it on.

**Comment by WilcoFiers** [comment]
Key question to be decided, will this rule be on by default?

**Comment by dan-tripp** [comment]
I'll try to fix this.

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base,
failing for the ratio:
`<div id="fail1" style="background-color: rgb(94, 5, 5); color: rgb(178, 21, 21); font-size: 20px; ">
<img width="1241" alt="image" src="https://user-images.githubusercontent.com/47409579/187343926-0efa2730-e0e9-44d1-9539-ce45f98f1668.png">

 and 
Passing for the ratio:

<img width="1198" alt="image" src="https://user-images.githubusercontent.com/47409579/187344110-9528190e-0b75-43bc-8abc-278776e5c914.png">
